### PR TITLE
chore: Add netlify deploy command list in cli and in export tab

### PR DESCRIPTION
**Pull request by JayaKrishnaNamburu**
## Description

Adds the `netlify` deployment commands sequence. Tested one by one, seems to be the best flow i can come by. Would like to validate on someone else too.

## Code Review

- [x] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [x] made a self-review

### CODE REVIEW

File: apps/builder/app/builder/features/topbar/publish.tsx [hunk 1]
```diff
@@ -492,7 +493,7 @@ const deployTargets = {
     docs: "https://vercel.com/docs/cli",
   },
   netlify: {
-    command: "npx netlify-cli",
+    command: `npx netlify-cli login\nnpx netlify-cli sites:create\nnpx netlify-cli build\nnpx netlify-cli deploy`,
```
Comment (kof) [inline apps/builder/app/builder/features/topbar/publish.tsx:496]: time to bring back the stripIndent utility or use array and join

File: packages/cli/src/commands/init-flow.ts [hunk 2]
```diff
@@ -107,9 +109,30 @@ export const initFlow = async (
...
       `Run ${pc.dim("npm run dev")} to preview your site on a local server.`,
-      `Run ${pc.dim("npx vercel")} to publish on Vercel.`,
+      projectTemplate && getDeploymentInstructions(projectTemplate),
     ]
       .filter(Boolean)
       .join("\n")
   );
 };
+
+const getDeploymentInstructions = (
```
Comment (kof) [inline packages/cli/src/commands/init-flow.ts:119]: why making it a function vs just a config?

**Review by kof — APPROVED** [review]

File: packages/cli/src/commands/init-flow.ts (hunk 2 above)
Comment (JayaKrishnaNamburu) [reply packages/cli/src/commands/init-flow.ts:119]: We have things like `netlify-functions` and `netlify-edge-functions` with same instructions. So, switch can make it easy to organise things around.

File: apps/builder/app/builder/features/topbar/publish.tsx [hunk 3]
```diff
@@ -492,7 +494,10 @@ const deployTargets = {
     docs: "https://vercel.com/docs/cli",
   },
   netlify: {
-    command: "npx netlify-cli",
+    command: `npx netlify-cli login
+npx netlify-cli sites:create
```
Comment (kof) [inline apps/builder/app/builder/features/topbar/publish.tsx:498]: personally I hate broken formatting

File: packages/cli/src/commands/init-flow.ts (hunk 2 above)
Comment (kof) [reply packages/cli/src/commands/init-flow.ts:119]: got it

File: packages/cli/src/commands/init-flow.ts [hunk 4]
```diff
@@ -107,9 +109,29 @@ export const initFlow = async (
...
+    case "netlify-edge-functions":
+      return [
+        `To deploy to Netlify, run the following commands: `,
+        `Run ${pc.dim("npx netlify-cli login")} to login to Netlify.`,
+        `Run ${pc.dim("npx netlify-cli sites:create")} to create a new site.`,
+        `Run ${pc.dim("npx netlify-cli build")} to build the site`,
+        `Run ${pc.dim("npx netlify-cli deploy")} to deploy on Netlify.`,
+      ].join("\n");
+    default:
+      return undefined;
```
[the remark this answers is not in the document]
Comment (kof) [reply packages/cli/src/commands/init-flow.ts:135]: did linter force you?

[the remark this answers is not in the document]
Comment (kof) [reply packages/cli/src/commands/init-flow.ts:135]: at least just `return;` kindof don't need to type it out

File: apps/builder/app/builder/features/topbar/publish.tsx (hunk 3 above)
Comment (JayaKrishnaNamburu) [reply apps/builder/app/builder/features/topbar/publish.tsx:499]: Done 👍

**Pull request merged**